python/ray/log_monitor.py

```diff
@@ -170,15 +172,38 @@ def check_log_files_and_publish_updates(self):
             assert not file_info.file_handle.closed
 
             lines_to_publish = []
+            driver_switches = []
```

Comment:
Please make this even clearer. It's not obvious to me from this comment that we need to add `1` to `index` to find such and such. The comment should be completely explicit so that people understand exactly how to use and manipulate the values in `driver_switches` without having to read the code.